[2025-09-21T11:36:12.606+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:35:00+00:00/task_id=validate_model_performance permission to 509
[2025-09-21T11:36:12.606+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:35:00+00:00/task_id=validate_model_performance permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:35:00+00:00/task_id=validate_model_performance'
[2025-09-21T11:36:12.647+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:35:00+00:00/task_id=validate_model_performance permission to 509
[2025-09-21T11:36:12.649+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:35:00+00:00/task_id=validate_model_performance permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:35:00+00:00/task_id=validate_model_performance'
[2025-09-21T11:36:12.715+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: train_predict_to_db.validate_model_performance scheduled__2025-09-21T11:35:00+00:00 [queued]>
[2025-09-21T11:36:12.735+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: train_predict_to_db.validate_model_performance scheduled__2025-09-21T11:35:00+00:00 [queued]>
[2025-09-21T11:36:12.736+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-09-21T11:36:12.762+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): validate_model_performance> on 2025-09-21 11:35:00+00:00
[2025-09-21T11:36:12.784+0000] {standard_task_runner.py:57} INFO - Started process 1164 to run task
[2025-09-21T11:36:12.791+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'train_predict_to_db', 'validate_model_performance', 'scheduled__2025-09-21T11:35:00+00:00', '--job-id', '100', '--raw', '--subdir', 'DAGS_FOLDER/train_predict_to_db_dag.py', '--cfg-path', '/tmp/tmp3rcs3zc9']
[2025-09-21T11:36:12.796+0000] {standard_task_runner.py:85} INFO - Job 100: Subtask validate_model_performance
[2025-09-21T11:36:12.882+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:35:00+00:00/task_id=validate_model_performance permission to 509
[2025-09-21T11:36:12.884+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:35:00+00:00/task_id=validate_model_performance permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:35:00+00:00/task_id=validate_model_performance'
[2025-09-21T11:36:12.887+0000] {task_command.py:416} INFO - Running <TaskInstance: train_predict_to_db.validate_model_performance scheduled__2025-09-21T11:35:00+00:00 [running]> on host 36da5371e73d
[2025-09-21T11:36:13.003+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='train_predict_to_db' AIRFLOW_CTX_TASK_ID='validate_model_performance' AIRFLOW_CTX_EXECUTION_DATE='2025-09-21T11:35:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-21T11:35:00+00:00'
[2025-09-21T11:36:13.048+0000] {python.py:194} INFO - Done. Returned value was: {'mse': 927282.0835252044, 'rmse': 962.9548709701844, 'mae': 756.8306662241617, 'r2': 0.9363852336036863}
[2025-09-21T11:36:13.091+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=train_predict_to_db, task_id=validate_model_performance, execution_date=20250921T113500, start_date=20250921T113612, end_date=20250921T113613
[2025-09-21T11:36:13.164+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-09-21T11:36:13.198+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
